src/tsx/liveScreen.tsx

```diff
@@ -12,28 +12,34 @@ interface IState {
   currentNode: string;
 }
 
-class LiveScreen extends React.PureComponent<{ coreConnection: ICoreConnection }, IState> {
+class LiveScreen extends React.PureComponent<
+  { coreConnection: ICoreConnection },
+  IState
+> {
   constructor(props: { coreConnection: ICoreConnection }) {
     super(props);
     this.state = { nodes: {}, currentNode: null };
     this.handleKey = this.handleKey.bind(this);
   }
 
-  public componentDidMount() {
+  public componentDidMount(): void {
     this.initConnection();
     document.addEventListener("keydown", this.handleKey);
   }
 
-  public componentWillUnmount() {
+  public componentWillUnmount(): void {
     document.removeEventListener("keydown", this.handleKey);
   }
 
-  public render() {
+  public render(): JSX.Element {
     return (
       <div className={style.screen}>
-        <StatusView/>
-        <Timeline nodes={this.state.nodes} currentNode={this.state.currentNode}/>
-        <PdfViewer/>
+        <StatusView />
+        <Timeline
```

Comment:
This is just an opinion and if there is some sort of community decision with a lot of experience behind it I won't complain, but it feels rather tedious to break up this on separate lines. I can see the reason why it's good for complex lines but it feels like it almost makes it harder to read for these short lines.
1. What do you think about this?
2. If we wanna keep short lines as everything-on-one-line, is it possible to conf the linter to it?

Reply:
1. I agree that short lines shouldn't be split up, but I'm not sure if this specific line is short enough. (I think it looks better on one line in github, but better split up in vscode. Not sure why.)
2. Prettier already doesn't split up lines below 80 characters, so if this had been 2 characters shorter it would have been on one line. This number is configurable, but they don't recommend increasing it: https://prettier.io/docs/en/options.html#print-width

Reply:
Let's leave it at 80 then :)